### PR TITLE
My account enhencement

### PR DESCRIPTION
**Pull request by ttaoS**
1. improve my account UI
2. handle dropdown value change
3. added static links in dropdown

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
 		"tslint": "^5.14.0",
 		"tslint-config-standard": "^8.0.1",
 		"tslint-react": "^4.0.0",
-		"typescript": "^3.4.1",
+		"typescript": "3.3.4000",
```
Comment (ttaoS) [inline package.json]: downgrade typescript as there is a performance issue with TS 3.4.1 working with styled component

File: src/ts/components/basic-components.ts [hunk 2]
```diff
@@ -3,6 +3,11 @@ import { colorSchemes } from './styles/themes'
 import { IAppState } from '../background/store/all'
 import { connect } from 'react-redux'
 import { Button as SemanticButton, Input } from 'semantic-ui-react'
+import Dropdown from 'semantic-ui-react/dist/commonjs/modules/Dropdown/Dropdown'
```
Comment (mytechtip) [inline src/ts/components/basic-components.ts]: How about simplify the import like:
```typescript
import { Dropdown, Image } from 'semantic-ui-react'
```

File: src/ts/components/basic-components.ts [hunk 3]
```diff
@@ -13,88 +18,114 @@ const mapStateToProps = (state: IAppState) => {
...
+  line-height: normal
+  letter-spacing: normal
+  color: #ffffff !important
+`
+
+export const DropdownItemIdenticon = styled(Identicon)`
+  display: inline;
+`
+
+export const StyledDropdownDivider = styled(DropdownDivider)`
```
Comment (mytechtip) [inline src/ts/components/basic-components.ts]: How about using `Dropdown.Divider` (if it works)?

File: webpack.config.js [hunk 4]
```diff
@@ -74,11 +74,6 @@ module.exports = {
 
 if (nodeEnv === 'watch') {
   module.exports.watch = true;
-  module.exports.watchOptions = {
```
Comment (mytechtip) [inline webpack.config.js]: Will it affect watch performance if removing those `ignored` configuration?

Comment (ttaoS) [reply webpack.config.js]: I added this before to improve performance but i don't think it makes diff. We already have include in tsconfig so it is bit redundancy. We still need to find out how we can improve dev build performance.

**Pull request merged**